app/messaging/process-message.js

```diff
@@ -1,47 +1,19 @@
-const util = require('util')
-const { set } = require('../repositories/application-repository')
-const { applicationResponseMsgType, applicationResponseQueue } = require('../config')
-const sendMessage = require('../messaging/send-message')
-const { notify: { templateIdApplicationComplete } } = require('../config')
-const sendEmail = require('../lib/send-email')
-const processApplication = async (msg) => {
-  const responseMessage = msg.body
-  let reference = ''
-  try {
-    // Get ID
-    const result = await set({
-      reference,
-      data: JSON.stringify(msg.body),
-      createdBy: 'admin',
-      createdAt: new Date()
-    })
-    // GetReference for ID
-    const application = result.dataValues
-    reference = msg.body.applicationId = application.reference
-    await sendMessage(msg.body, applicationResponseMsgType, applicationResponseQueue, { sessionId: msg.body.sessionId })
-  } catch {
-    responseMessage.error = {
-      message: 'can\'t submit application at this time.'
-    }
-  } finally {
-    const result = await sendEmail(templateIdApplicationComplete, msg.body.organisation.email, { personalisation: { name: msg.body.organisation.name, reference }, reference })
-
-    if (!result) {
-      responseMessage.error = {
-        message: 'can\'t send message.'
-      }
-    }
-  }
-  return responseMessage
-}
+const { applicationRequestMsgType, fetchApplicationRequestMsgType } = require('../config')
+const fetchApplication = require('./fetch-application')
+const processApplication = require('./process-application')
 
 const processApplicationMessage = async (message, receiver) => {
   try {
-    const { body: msgBody } = message
-    console.log('Application received:', util.inspect(msgBody, false, null, true))
-    await processApplication(message)
+    const { applicationProperties: properties } = message
+    if (properties.type === fetchApplicationRequestMsgType) {
```

Comment:
Maybe change the `if` blocks to a `switch` as each message should only have a single type.